ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/JsonRpcMethodsFactory.java

```diff
@@ -139,6 +146,8 @@
 
   public Map<String, JsonRpcMethod> methods(
       final String clientVersion,
+      final int networkId,
+      final JsonNode genesisConfigConfig,
```

Comment:
genesisConfigConfig?

Reply:
The config element of the genesisConfig.